.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+Please open an issue for this pull request on Github Issues as well.
+https://github.com/intel/e2eAIOK/issues
+Pull Request Name format: [${VERSION_ID}][ISSUE-${ISSUES_ID}] ${detailed message}
+ex: [v1.1][ISSUE-190] Add PR to issue link
+
+  1. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][VERSION_ID][ISSUE_ID] Your PR title ...'.
+  2. Be sure to keep the PR description updated to reflect all changes.
+  3. Please write your PR title to summarize what this PR proposes.
+-->
+
+### What changes were proposed in this pull request?
+<!--
+Please clarify what changes you are proposing. The purpose of this section is to outline the changes and how this PR fixes the issue. 
+-->
+
+
+### Why are the changes needed?
+<!--
+Please clarify why the changes are needed. For instance,
+  1. If you propose a new API, clarify the use case for a new API.
+  2. If you fix a bug, you can clarify why it is a bug.
+-->
+
+
+### Does this PR introduce _any_ user-facing change?
+<!--
+If yes, please clarify the change.
+-->
+
+
+### How was this patch tested?
```

Comment:
assuming most of the PR will be covered by github workflows automatically, do we need this?

Reply:
@Jian-Zhang , by adding this, we can make sure all path will be tested. Committer can either point to a pre-defined UT or add new UT to cover this new PR, Or simply say this is document fix, no tests required.
So we won't add new function out UT coverage.